### PR TITLE
Testing String overrides

### PR DESCRIPTION
**Pull request by joyceg**
Php Unit test for string overrides.

### CODE REVIEW

**Comment by naveenvalecha** [comment]
@joyceg The PR needs updation.

**Comment by joyceg** [comment]
Drupal/search_config/ usage was removed in the earlier [commit](https://github.com/naveenvalecha/search_config/pull/16/commits/a2f061f29ae7cc45b5fb55828f48633dd6645f42v).

File: tests/src/Unit/stringOverridesTest.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Testing the string overrides: search_config_string_overrides()
+ */
+class stringOverridesTest extends PHPUnit_Framework_TestCase {
```
Comment (naveenvalecha) [inline tests/src/Unit/stringOverridesTest.php]: have you missed the use \ PHPUnit_Framework_TestCase  statement ?

File: tests/src/Unit/stringOverridesTest.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Testing the string overrides: search_config_string_overrides()
+ */
```
Comment (naveenvalecha) [inline tests/src/Unit/stringOverridesTest.php:5]: add the group here.

File: tests/src/Unit/stringOverridesTest.php (hunk 1 above)
Comment (naveenvalecha) [inline tests/src/Unit/stringOverridesTest.php]: this would be \ PHPUnit_Framework_TestCase.

**Pull request merged**